packages/hub/main.ts

```diff
@@ -250,7 +260,7 @@ declare module '@cardstack/di' {
   }
 }
 
-function initSentry() {
+export async function initSentry() {
```

Comment:
is this really async? where's the `await`?

Reply:
oops, Copilot mistake I think 🙃 removed now thanks!